apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -316,9 +323,9 @@ websocket_handle({text, MsgBin}, _ConnState, #state{regs=Register, role=Role}=St
         _ ->
             ct:log("[~p] Received msg ~p~n", [Role, Msg])
     end,
-    Origin = binary_to_existing_atom(maps:get(<<"origin">>, Msg,
-                                              atom_to_binary(?CHANNEL, utf8)), utf8),
-    Action = binary_to_existing_atom(maps:get(<<"action">>, Msg), utf8),
+    Origin = binary_to_atom(maps:get(<<"origin">>, Msg,
```

Comment:
I see this is test only so no concern for dynamic creation of atoms.

Reply:
Correct. The change was made after a test case failure, and I thought it was better to not have to explicitly create atoms in the test suite just to ensure that it didn't crash on `binary_to_existing_atom/2`.